src/renderers/webgl/WebGLTextures.js

```diff
@@ -59,11 +59,21 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 				canvas.height = height;
 
 				var context = canvas.getContext( '2d' );
+
+				// ImageBitmap is flipped vertically
+
+				if ( useOffscreenCanvas ) {
```

Comment:
This block of code is not needed now because we're are now returning the canvas.

Reply:
Please ignore what I said about the issue disappearing. I got confused. What fixed it was just returning the `canvas` instead of the `ImageBitmap`.